src/grpc-unary-method.js

```diff
@@ -26,9 +26,15 @@ class GrpcUnaryMethod extends GrpcMethod {
     let request
 
     try {
-      this.logRequestStart()
+      const { method, auth, createLogger, requestOptions } = this
+
+      // generate unique id to be associated with the request
+      const requestId = generateId(6)
+
+      // create the logger with the requestId and messageid
+      const logger = createLogger({ messageId: this.messageId, requestId })
```

Comment:
why not pass the entire request object in here? It might be nice to have the `messageId` on the request object too.

Reply:
The request object contains the logger? How can the logger take in the request object? Unless I am misunderstanding. Do you mean the call.request?

Reply:
oh right, I forgot the request object contains the logger. I was thinking you could pass `request` in here, but it would require instantiating the `request` object, passing to `createLogger` to create `logger` and then modifying `request` to have `logger` as a property. Not sure if it's worth it - your call.